packages/x-components/src/design-system/components/picture/fixed-ratio.scss

```diff
@@ -1,19 +1,5 @@
 .x-picture--fixed-ratio.x-picture {
-  // layout
-  position: relative;
-  height: 0;
-
   // size
-  padding-block-start: calc(var(--x-number-aspect-ratio-picture) * 100%);
-  padding-block-end: 0;
-  padding-inline-start: 0;
-  padding-inline-end: 0;
+  aspect-ratio: 1 / var(--x-number-aspect-ratio-picture);
```

Comment:
Wasn't the agreement to use just the `var(--x-number-aspect-ratio-picture)` here and marking the merge as BREAKING CHANGE?